tests/view_information_about_new_person/test_current_view_person_info.py

```diff
@@ -0,0 +1,124 @@
+# coding=utf-8
+import time
+
+__author__ = 'Vadym'
+
+from model.user import User
+
+def test_global_info_about_person(app):
+    app.ensure_logout()
+    app.login(User.Admin())
+    app.internal_page.is_this_page
+    app.internal_page.driver.get('http://194.44.198.221/#/person/view/68/main')
+    time.sleep(8)
+
+    fio_ukranian = app.person_current_view_page.get_fio_ukranian
+    assert fio_ukranian
+    assert u'Малкович Малкович Малкович'.encode("cp1251") == fio_ukranian.text.encode("cp1251")
+
+    fio_english = app.person_current_view_page.get_fio_english
+    assert fio_english
+    assert 'Malkovich Malkovich' == fio_english.text
+
+    assert 'MA' == app.person_current_view_page.get_serial_person_record()
+
+    assert '19531209' == app.person_current_view_page.get_number_person_record()
+
+
+def test_main_info_about_person(app):
+    data_main_info_person = [u'абітурієнт', u'09.12.1953 р.', u'✘', u'Чоловіча',
```

Comment:
it is not so good to depend on the indexes of the items in this array. can you propose something better?